lib/bolt_ext/server.rb

```diff
@@ -22,7 +22,8 @@ class TransportAPI < Sinatra::Base
     content_type :json
 
     body = JSON.parse(request.body.read)
-    schema = "lib/bolt_ext/schemas/ssh-run_task.json"
+    schema = File.join(__dir__, 'schemas', 'ssh-run_task.json')
+    JSON::Validator.add_schema(shared_schema)
```

Comment:
Is this thread-safe? Maybe we should just do this on startup.

Reply:
I'd like to see this load all the schemas into memory at startup (the `initialize` function), add the shared schema once, then use the preloaded ssh and winrm schemas to validate the body.